src/lock.js

```diff
@@ -19,7 +19,7 @@ exports.lock = (dir, callback) => {
   const file = path.join(dir, lockFile)
   log('locking %s', file)
 
-  lock(dir, {lockfilePath: file})
+  lock(dir, {lockfilePath: file, stale: 20000})
```

Comment:
`20000` needs to be pulled out as a const with a comment explaining what it does and why it is set to 2000 with a link to this PR.